packages/web3-modal/src/components/modals/TransactionsModal/styled.tsx

```diff
@@ -10,6 +10,7 @@ import {
   ThumbsUp
 } from '@past3lle/components'
 import React from 'react'
+import { BackgroundStyles } from 'src/theme/types'
```

Comment:
fix: impot path needs to be relative e.g `../theme/types`

Reply:
fixed: [7454c1a](https://github.com/PAST3LLE/monorepo/pull/77/commits/7454c1aa18319a4bf41a6e5296dde092e9e5470f)